fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,4 +1,4 @@
-Dicio is a <i>free and open source</i> <b>voice assistant</b>. It supports many different <b>skills</b> and input/output methods, and it provides both <b>speech</b> and <b>graphical</b> feedback to a question. It uses <a href="https://github.com/alphacep/vosk-api/">Vosk</a> for <i>speech to text</i>. It has <b>multilanguage</b> support, and is currently available in Czech, English, French, German, Greek, Italian, Polish, Russian, Slovenian, Spanish, Swedish and Ukrainian.
+Dicio is a <i>free and open source</i> <b>voice assistant</b>. It supports many different <b>skills</b> and input/output methods, and it provides both <b>speech</b> and <b>graphical</b> feedback to a question. It uses <a href="https://github.com/alphacep/vosk-api/">Vosk</a> for <i>speech to text</i>. It has <b>multilanguage</b> support, and is currently available in Czech, English, French, German, Greek, Italian, Polish, Russian, Slovenian and Spanish, Ukrainian.
```

Comment:
Don't remove languages and/or other skills from this file (I guess this was a rebase issue)